.rubocop.yml

```
@@ -7,5 +7,9 @@ inherit_gem: { rubocop-rails-omakase: rubocop.yml }
 # Layout/SpaceInsideArrayLiteralBrackets:
 #   Enabled: false
 
-Style/FrozenStringLiteralComment:
-  Enabled: true
+inherit_gem:
```

YAMLlint (1.37.1)

[error] duplication of key "inherit_gem" in mapping

(key-duplicates)